hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -712,4 +847,421 @@ static Throwable peel(Throwable e) {
+                    break;
+                case INTEGER:
+                    colWidths[i] = Math.max(SQLCliConstants.INTEGER_FORMAT_LENGTH,
+                            Math.min(colName.length(), SQLCliConstants.VARCHAR_FORMAT_LENGTH));
+                    break;
+                case NULL:
+                case TINYINT:
+                    colWidths[i] = Math.max(SQLCliConstants.TINYINT_FORMAT_LENGTH,
+                            Math.min(colName.length(), SQLCliConstants.VARCHAR_FORMAT_LENGTH));
+                    break;
+                case SMALLINT:
+                    colWidths[i] = Math.max(SQLCliConstants.SMALLINT_FORMAT_LENGTH,
+                            Math.min(colName.length(), SQLCliConstants.VARCHAR_FORMAT_LENGTH));
+                    break;
+                case TIMESTAMP:
+                    colWidths[i] = Math.max(SQLCliConstants.TIMESTAMP_FORMAT_LENGTH,
+                            Math.min(colName.length(), SQLCliConstants.VARCHAR_FORMAT_LENGTH));
+                    break;
+                case BIGINT:
+                case VARCHAR:
+                case OBJECT:
+                default:
+                    colWidths[i] = SQLCliConstants.VARCHAR_FORMAT_LENGTH;
+            }
+        }
+        return colWidths;
+    }
+
+    private static Alignment[] determineAlignments(SqlRowMetadata metadata) {
+        int colCount = metadata.getColumnCount();
+        Alignment[] alignments = new Alignment[colCount];
+        for (int i = 0; i < colCount; i++) {
+            SqlColumnMetadata colMetadata = metadata.getColumn(i);
+            SqlColumnType type = colMetadata.getType();
+            String colName = colMetadata.getName();
+            switch (type) {
+                case BIGINT:
+                case DECIMAL:
+                case DOUBLE:
+                case INTEGER:
+                case REAL:
+                case SMALLINT:
+                case TINYINT:
+                    alignments[i] = Alignment.RIGHT;
+                    break;
+                case BOOLEAN:
+                case DATE:
+                case NULL:
+                case OBJECT:
+                case TIMESTAMP:
+                case VARCHAR:
+                case TIMESTAMP_WITH_TIME_ZONE:
+                default:
+                    alignments[i] = Alignment.CENTER;
+            }
+        }
+        return alignments;
+    }
+
+    private static void printMetadataInfo(SqlRowMetadata metadata, int[] colWidths,
+                                          Alignment[] alignments, PrintWriter out) {
+        int colCount = metadata.getColumnCount();
+        printSeparatorLine(colCount, colWidths, out);
+        AttributedStringBuilder builder = new AttributedStringBuilder()
+                .style(AttributedStyle.BOLD.foreground(SECONDARY_COLOR));
+        builder.append("|");
+        for (int i = 0; i < colCount; i++) {
+            String colName = metadata.getColumn(i).getName();
```

Comment:
We should sanitize the `colName` the same way we sanitize values on line 1118 below. Or even more, replace every character below 32 with `.` or something.